.idea/codeStyles/Project.xml

```diff
@@ -11,5 +11,8 @@
       <option name="HTML_DO_NOT_INDENT_CHILDREN_OF" value="html,body" />
       <option name="HTML_ENFORCE_QUOTES" value="true" />
     </HTMLCodeStyleSettings>
+    <JSCodeStyleSettings version="0">
+      <option name="SPACES_WITHIN_OBJECT_LITERAL_BRACES" value="true" />
+    </JSCodeStyleSettings>

```

Comment:
Hmmm - I thought we were suppose to exclude .idea files?

Reply:
Why?

Reply:
Cause it is in gitignore 😂 

Just not quite right ;) we should make it so all files in idea are ignored, not just the first level of them.

Reply:
On a more serious note, out choice of linting rules lives in non-tool specific tools, like rubocop.

Reply:
Yes, but why is commiting this a bad thing? Eslint should complain about this regardless, but it makes sense for me for all devs using RubyMine to have it just work

Reply:
My guess is that it's because they tend to be a messy, unreadable xml gabage. And, if one person changes their IDE settings, then everyone on the project has to too, or else! 

And those settings include a lot of cosmetic things, or preferences rather than hard rules. Do you like your IDE in light mode? No? Though shit, I just committed an IDE change to make it dark 😂

Reply:
https://intellij-support.jetbrains.com/hc/en-us/articles/206544839-How-to-manage-projects-under-Version-Control-Systems

Reply:
Also (perish the thought) what if two people use different IDEs that both use `.idea` directory to store settings and they are not fully compatible with each other? ooof, I don't even want to think about it :P

Reply:
can i add my vim config to the repo?

Reply:
I don't think it needs to be done in this PR, but we should probably add a ticket for 'sorting out idea files'. Maybe after talking it through on a dev catchup.